docker-build-v2/amd64-linux/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM docker.io/ubuntu:18.04
```

Comment:
```suggestion
# You can't use the build on an older Linux system due to system libs linkage,
# so pick a relatively old one instead of freshest so that everybody can play.
FROM docker.io/ubuntu:18.04
```

Reply:
Adjusted text slightly and commited.